### PR TITLE
Client db

### PR DESCRIPTION
**Pull request by mw1510**
Let me know if you want the format to change for anything.

### CODE REVIEW

File: backend/models/clients.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose')
+
+const clientSchema = new mongoose.Schema({
```
Comment (HariCodesThings) [inline backend/models/clients.js:3]: Schema looks good! 
We should add extra information regarding what fields are required (expecting non-null) and default values. Please see Task 4 on the Sprint 2 page (https://www.notion.so/h4i/Sprint-2-44ae536312bf4c71a0057500cdd15ad7) for updated information on required/default fields.

To include this code in mongoose, you simply need to edit the attributes as follows:
`attributeName: {type: type, required: true, default: defaultVal}`

Let me know if you run into trouble with this, and we can troubleshoot.

Comment (mw1510) [reply backend/models/clients.js:3]: Alright I updated the schema. Let me know if I forgot to add something or something needs to be changed.

Comment (HariCodesThings) [reply backend/models/clients.js:3]: Thanks for making the changes. It's super close now, just that there should be a `required: true` on `foodDays`. Good to go aside from that.

**Comment by mw1510** [comment]
Just finished adding the required for food days

**Review by EmilyGavrilenko — APPROVED** [review]
Everything looks great, good work!

**Pull request merged**